fastlane/lanes/release_management_in_ci.rb

```diff
@@ -75,4 +75,39 @@
       environment: { RELEASE_VERSION: release_version }
     )
   end
+
+  lane :trigger_new_hotfix_in_ci do |options|
+    version = extract_hotfix_version_from_lane_options!(options)
+
+    buildkite_trigger_build(
+      buildkite_organization: BUILDKITE_ORGANIZATION,
+      buildkite_pipeline: BUILDKITE_PIPELINE,
+      branch: compute_release_branch_name(options:, version:),
```

Comment:
I know this is not the Ruby style we agreed upon, but the linter configuration is not set to pick this up. I decided to roll with it and delay addressing it till we'll have the linter to flag this for us.

Reply:
Also, the branch used here is irrelevant because the hotfix automation will checkout the base tag and create a new branch from it.